styles/header.css

```diff
@@ -32,23 +41,41 @@
   text-indent: 100%;
   white-space: nowrap;
   width: 250px;
+
+  &.polychrome {
+    background-color: transparent;
+    mask-image: none;
+    background-image: url("StanfordLibraries-logo-poly.svg");
+    background-repeat: no-repeat;
+    background-position: 0 center;
+  }
 }
 
 /* The .navbars selector is because adding .navbar to the nav in the masthead
  * causes popper to incorrectly position the dropdowns offscreen, so we need
  * to target that navigation with a different selector. */
-.bg-dark.navbar,
-.bg-dark .navbar,
-.bg-dark .navbars {
-  --bs-navbar-color: white;
-  --bs-navbar-active-color: var(--bs-navbar-color);
-  --bs-navbar-hover-color: rgb(255 255 255 / 100%);
-  --bs-nav-link-color: var(--bs-navbar-color);
-  --bs-navbar-toggler-icon-bg: url("data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 17 17'%3e%3cpath stroke='rgba%28255, 255, 255, 1%29' stroke-linecap='round' stroke-miterlimit='10' stroke-width='1' d='M2.146 2.854a.5.5 0 1 1 .708-.708L8 7.293l5.146-5.147a.5.5 0 0 1 .708.708L8.707 8l5.147 5.146a.5.5 0 0 1-.708.708L8 8.707l-5.146 5.147a.5.5 0 0 1-.708-.708L7.293 8z' /%3e%3c/svg%3e");
-  --bs-navbar-toggler-border-color: rgb(255 255 255 / 10%);
-
-  .collapsed {
-    --bs-navbar-toggler-icon-bg: url("data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 30 30'%3e%3cpath stroke='rgba%28255, 255, 255, 1%29' stroke-linecap='round' stroke-miterlimit='10' stroke-width='2' d='M4 7h22M4 15h22M4 23h22'/%3e%3c/svg%3e");
+.navbar,
+.navbars {
+  --bs-navbar-color: var(--stanford-black);
+  --bs-nav-link-color: var(--stanford-black);
+  --bs-link-color-rgb: var(--stanford-black-rgb);
+  --bs-link-hover-color-rgb: var(--stanford-black-rgb);
+  --bs-light-rgb: var(--stanford-10-black-rgb);
+
+  &.bg-dark,
+  .bg-dark & {
```

Comment:
Why do we need both of these variants? Can you explain with a comment in the code?

Reply:
I think there is going to need to a be a version of this because the color for text is different between variants but currently some of this code isn't being used (and needs to be) for the light variants. There is missing functionality that when the menu is open, the hamburger should become a X. This is only happening in the existing headers not the variants.

![Screenshot 2025-01-31 at 3 15 47 PM](https://github.com/user-attachments/assets/3c2f3da9-31cb-49b4-8f29-15ee4cf03009)
![Screenshot 2025-01-31 at 3 15 41 PM](https://github.com/user-attachments/assets/9324c6f8-dfe5-4e58-b0e3-eaedbcced2df)

Reply:
Found a way to not need both selectors (by adding a class to the HTML) that also resolved some other issues.